### PR TITLE
Add config params to flags

### LINKED ISSUE #3: Add config options and cmd line flags
# Rationale

Right now there are some values that are just hardcoded but that a user should be able to tweak:
- Supported EntryPoint addresses
- Account to be used for signing transactions
- Maximum allowed user ops a sender can have in the pool at a given time
- Maximum number of mev bundles that should be created when bundling user operations

### PR DESCRIPTION
**Pull request by jmederosalvarado**
Closes #3

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -374,6 +374,26 @@ var (
 		Usage: "Maximum amount of time non-executable transaction are queued",
 		Value: ethconfig.Defaults.TxPool.Lifetime,
 	}
+	// User operation pool settings
+	UserOpBundlingEnabledFlag = cli.BoolFlag{
+		Name:  "uopool.bundle",
```
Comment (kristofgazso) [inline cmd/utils/flags.go:379]: uo or op as a shortening?

File: core/userop_pool.go [hunk 2]
```diff
@@ -219,25 +162,47 @@ func (pool *UserOpPool) mainLoop() {
...
-	close(pool.stopCh)
+	pool.stopCh <- struct{}{}
 
 	// Wait for workers to stop
 	pool.wg.Wait()
 
 	log.Info("UserOp pool stopped")
 }
 
+func (pool *UserOpPool) SetSigner(wallet accounts.Wallet, account accounts.Account) {
```
Comment (kristofgazso) [inline core/userop_pool.go:189]: should these things be public?

File: cmd/utils/flags.go (hunk 1 above)
Comment (jmederosalvarado) [reply cmd/utils/flags.go:379]: hmm, hadn't thought about that, I think I actually line op better

File: core/userop_pool.go (hunk 2 above)
Comment (jmederosalvarado) [reply core/userop_pool.go:189]: Yeah, they are used in `cmd/geth` to start/modify the bundling process using command line flags and config options. it is the same as the `SetEtherbase` in the miner.

**Pull request merged**